src/App.css

```diff
@@ -8,13 +8,14 @@
 }
```

Comment:
One thing that would be good for you to start doing is to create a separate css files for each of your React components and putting the related classes in that.

E.g. for the `Heading.js` component, create a `Heading.css` file and put the `logo` and `.app-header` classes into that and import that file into `Heading.js`
```
import "./Heading.css";
```
Why do this? That way you don't end up with one huge css file that contains everything. It's also easier to find which classes apply to which components.